pkgs/by-name/gi/gi-docgen/package.nix

```diff
@@ -27,7 +28,7 @@ python3.pkgs.buildPythonApplication rec {
     ninja
   ];
 
-  pythonPath = with python3.pkgs; [
+  propagatedBuildInputs = with python3.pkgs; [
```

Comment:
I believe the [comment](https://github.com/NixOS/nixpkgs/commit/3e05cce97ca3ea9eee42a17c8bc0b89345b5d0b6) predates `dependencies` attribute. Also we should double check that this does not pollute dependents’ build environment (what we are trying in `postFixup`).

Reply:
I do not know off the top of my head. Can you check the propagated build inputs file content?